samples/armv6-rpi-linux-gnueabihf/crosstool.config

```diff
@@ -0,0 +1,25 @@
+CT_LOCAL_TARBALLS_DIR="${HOME}/src"
+CT_SAVE_TARBALLS=y
+CT_LOG_EXTRA=y
+CT_ARCH_CPU="arm1176jzf-s"
+CT_ARCH_FPU="vfp"
+CT_ARCH_FLOAT_HW=y
+CT_ARCH_arm=y
+CT_ARCH_SUFFIX="v6"
+CT_TARGET_VENDOR="rpi"
+CT_KERNEL_linux=y
+CT_KERNEL_V_4_1=y
+CT_BINUTILS_LINKER_LD_GOLD=y
+CT_BINUTILS_GOLD_THREADS=y
+CT_BINUTILS_LD_WRAPPER=y
+CT_BINUTILS_PLUGINS=y
+CT_LIBC_LOCALES=y
+CT_LIBC_GLIBC_KERNEL_VERSION_CHOSEN=y
+CT_LIBC_GLIBC_MIN_KERNEL_VERSION="3.2.27"
```

Comment:
Stéphane,

Explain why you kept glibc support for that kernel:  that's the oldest kernel
with support for the RPi-1, so no need to have backward compatibility with
older kernels.